csp/settings.py

```diff
@@ -210,3 +210,90 @@
 DROPBOX_APP_KEY = '__KEY__'
 DROPBOX_APP_SECRET = '__SECRET__'
 DROPBOX_REDIRECT_URI = 'http://localhost:8000/api/dropbox-auth-finish'
+
+# Secure Settings
+if not DEBUG:
+    # Honor the 'X-Forwarded-Proto' header for request.is_secure()
+    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+    SECURE_HSTS_SECONDS = 31536000
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+    SECURE_FRAME_DENY = True
+    SECURE_CONTENT_TYPE_NOSNIFF = True
+    SECURE_BROWSER_XSS_FILTER = True
+    SESSION_COOKIE_SECURE = False
+    CSRF_COOKIE_SECURE = True
+    SESSION_COOKIE_HTTPONLY = True
+    SECURE_SSL_REDIRECT = True
+
+
+# MANAGER CONFIGURATION
+# ------------------------------------------------------------------------------
+# See: https://docs.djangoproject.com/en/dev/ref/settings/#admins
+ADMINS = (
+    ("Shirish Goyal", 'shirish.goyal@gmail.com'),       # add more team members
```

Comment:
We can use daemo@cs.stanford.edu, which autoforwards to Slack.

On Wed, Sep 2, 2015 at 3:59 PM, Rohit <notifications@github.com> wrote:

> In csp/settings.py
> <https://github.com/crowdresearch/crowdsource-platform/pull/476#discussion_r38595678>
> :
>
> > +    SECURE_HSTS_SECONDS = 31536000
> > +    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
> > +    SECURE_FRAME_DENY = True
> > +    SECURE_CONTENT_TYPE_NOSNIFF = True
> > +    SECURE_BROWSER_XSS_FILTER = True
> > +    SESSION_COOKIE_SECURE = False
> > +    CSRF_COOKIE_SECURE = True
> > +    SESSION_COOKIE_HTTPONLY = True
> > +    SECURE_SSL_REDIRECT = True
> > +
> > +
> > +# MANAGER CONFIGURATION
> > +# ------------------------------------------------------------------------------
> > +# See: https://docs.djangoproject.com/en/dev/ref/settings/#admins
> > +ADMINS = (
> > +    ("Shirish Goyal", 'shirish.goyal@gmail.com'),       # add more team members
>
> let's set this to the group or the general alias?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crowdresearch/crowdsource-platform/pull/476/files#r38595678>
> .
>